pkgs/development/rocm-modules/6/llvm/base.nix

```diff
@@ -86,7 +86,7 @@ in stdenv.mkDerivation (finalAttrs: {
     cmake
     ninja
     git
-    python3Packages.python
+    (python3Packages.python.withPackages (p: [ p.setuptools ]))
```

Comment:
What bothers me a bit is that recommonmark used a bit down below is not included in this environment. But if it works, I guess it is OK.

Reply:
Yeah, the first iteration of the PR was including it similar to that package, I dont understand why this works and that didn't...